docs/integrations/airflow/operator.md

```diff
@@ -1,50 +1,189 @@
 ---
-sidebar_position: 2
+sidebar_position: 3
 ---
 
-# Lineage in custom operators
+# Exposing lineage in Airflow operators
 
-As a way of extending the OpenLineage Airflow integration, we are providing a way to [add custom extractors](./extractor.md). 
-This is great for people who want to add integration with external operators - for example, a regular Airflow user who wants to support, for example, `GCSToS3Operator`.
+Since: 0.18.0
 
-However, it's too cumbersome for people who own operators, and want to add the default implementation of OpenLineage for their operators for external users. 
-This is why we have an additional way to expose lineage: the `get_openlineage_facets_on_start` and `get_openlineage_facets_on_complete` methods that can be implemented on your Operator.<br />When any of those are present, OpenLineage's `DefaultExtractor` uses them to get lineage data. If you don't define `get_openlineage_facets_on_complete` it will run `get_openlineage_facets_on_start` instead.
+## Scope
+Lineage extraction logic should be as close as possible to the operator logic. (for airflow operators in the airflow repository, for other providers in their own repository). This makes lineage extraction more stable as it lives with the operators. 
```

Comment:
"Lineage extraction logic should be as close as possible to the operator logic. For Airflow's included operators, this logic would ideally live in the Airflow repository; for external providers, it would live in their own repository. This makes lineage extraction more stable, as it lives with the operators."